forge-gui/res/cardsfolder/upcoming/mishras_domination.txt

```diff
@@ -0,0 +1,8 @@
+Name:Mishra's Domination
```

Comment:
This might need some kind of AI tag to help the AI know what to do with it

@Agetian any thoughts?